articles/azure/azs-how-use-azure-api.md

```diff
@@ -112,36 +136,54 @@ To obtain an access token:
       "ext_expires_in": "0",
       "expires_on": "1512574780",
       "not_before": "1512570880",
-      "resource": "https://management.ukcloud.onmicrosoft.com/c0a3a144-57ae-42e2-a5da-19e945a25deb",
-      "access_token": "eyJ0eXAiOi...truncated for readability..."
+      "resource": "https://management.as2ukcloud.onmicrosoft.com/90ada28c-5aed-4248-90c7-0538504217f1",
+      "access_token": "eyJ0eXAiOi...truncated for readability...",
+      "refresh_token": "AQABAAAAAA...truncated for readability..."
     }
     ```
 
-5. You must include this token in the Authorization header of each subsequent API request. For example:
+5. You must include this token in the Authorization header of each subsequent API request, for example:
+
+    ### [Bash](#tab/tabid-3)
 
     ```bash
-    curl -H "Authorization: Bearer eyJ0eXAiOi...truncated for readability..." 'https://management.local.azurestack.external/subscriptions?api-version=2016-05-01'
+    curl -H "Authorization: Bearer eyJ0eXAiOi...truncated for readability..." 'https://management.frn00006.azure.ukcloud.com/subscriptions?api-version=2017-12-01'
+    ```
+
+    ### [PowerShell](#tab/tabid-4)
+
+    ```powershell
+    # Add access token to header object for subsequent API requests
+    $AuthHeader = @{"Authorization" = "Bearer $($AuthResp.access_token)"}
+
+    # Query the Azure Stack Hub API for subscriptions
+    Invoke-RestMethod -Method "GET" -Uri "https://management.frn00006.azure.ukcloud.com/subscriptions" -Headers $AuthHeader -Body @{"api-version" = "2017-12-01"} -ContentType "application/x-www-form-urlencoded"
     ```
 
+    ***
+
 ## Calling Azure Stack Hub API endpoints
 
 A REST request URI consists of:
 
-`<URI-scheme>://<URI-host>/<resource-path>?<query-string>`
+   `<URI-scheme>://<URI-host>/<resource-path>?<query-string>`
 
 where:
 
-- `URI-scheme` is the protocol used to transmit the request, for example `http` or `https`.
+- **URI-scheme** is the protocol used to transmit the request, for example `http` or `https`
+
+- **URI-host** is the domain name or IP address of the server where the REST service endpoint is hosted, for example: `management.frn00006.azure.ukcloud.com`
 
-- `URI-host` is the domain name or IP address of the server where the REST service endpoint is hosted, for example `management.local.azurestack.external`.
+- **resource-path** is the resource or resource collection, which may include multiple segments used by the service in determining the selection of those resources, for example: `/subscriptions` is the resource path to obtain information about Azure Stack Hub subscriptions
```

Comment:
Think it might be clearer for this to be:
resource path is the resource or resource collection used by the service in determining the selection of those resources (the path may include multiple segments), for example...

Reply:
Or just add an extra comma:
resource-path is the resource or resource collection, which may include multiple segments, used by the service...